docs/en/application.md

```diff
@@ -1,5 +1,5 @@
 ---
-title:  Application CRD
+title: Deploy Application
```

Comment:
This doc need rewrite, it should only about deploy, no definitions and "contracts" should be included.